### PR TITLE
Features/map markers

### PR DESCRIPTION
**Pull request by NewIvanIvanov**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,31 +1,46 @@
...
 pytz==2017.3
+singledispatch==3.4.0.3
 six==1.11.0
 sqlacodegen==1.1.6
 SQLAlchemy==1.1.14
 SQLAlchemy-Utils==0.32.19
 visitor==0.1.3
 Werkzeug==0.12.2
+wrapt==1.10.11
 WTForms==2.1
```
Comment (KostSvitlana) [inline requirements.txt]: you added a lot of  packages. They exactly needded?

**Comment by KostSvitlana** [comment]
Need resolve conflicts

File: requirements.txt (hunk 1 above)
Comment (NewIvanIvanov) [reply requirements.txt]: main part of this packages are nedded for pylint

File: client/apps/city_issues/static/city_issues/js/main.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  IssueMap.prototype.setViewPoint = function(latitude, longitude, scale) {
+    this.map.setView([latitude, longitude], scale);
+  };
+
+
+  IssueMap.prototype.addMapLayer = function(mapLink, mapZoom, mapAttribute) {
+    L.tileLayer(mapLink, {maxZoom: mapZoom, attribution: mapAttribute}).addTo(this.map);
+  };
+
+  IssueMap.prototype.chooseMaker = function(color) {
```
Comment (serge30) [inline client/apps/city_issues/static/city_issues/js/main.js]: Probably `chooseMarker`?

File: backend/models/issues.py [hunk 3]
```diff
@@ -53,8 +53,9 @@ class Issue(db.Model):
     user_id = db.Column(db.ForeignKey(u'users.id'), index=True)
     category_id = db.Column(db.ForeignKey(
         u'category.id'), nullable=False, index=True)
+    locationX = db.Column(db.Float)
+    locationY = db.Column(db.Float)
```
Comment (serge30) [inline backend/models/issues.py]: I'm not sure it's a good names for these fields. First at all the are not follow naming style, secondly names are not reflect what they are store. I'd suggest to use something like `loacation_lon` and `location_lat`.

**Review by KostSvitlana — APPROVED** [review]

File: backend/config.py.example [hunk 4]
```diff
@@ -1,22 +1,20 @@
-# Rename this file to config.py and fill SQLALCHEMY_DATABASE_URI with your's data.
+ Rename this file to config.py and fill db_credentials with your's data.
+import os
```
Comment (serge30) [inline backend/config.py.example]: I don't see using of `os` module here.

File: backend/config.py.example [hunk 5]
```diff
@@ -1,22 +1,20 @@
...
+
 
 class Config(object):
+    # Change database credentials to yours.
+    db_credentials = 'postgres://user:password@localhost/db_name'
```
Comment (serge30) [inline backend/config.py.example]: If you want to use separate config variable for DB connection string, please make it naming style according to variables below.

File: client/apps/city_issues/views.py [hunk 6]
```diff
@@ -3,8 +3,27 @@
 """
 # -*- coding: utf-8 -*-
 from django.views.generic.base import TemplateView
+from django.shortcuts import render
+from django.http import JsonResponse
+from django.core import serializers
+from city_issues.models import Issues
```
Comment (serge30) [inline client/apps/city_issues/views.py]: Imports are not separated by groups and are not sorted.

**Review by serge30 — CHANGES_REQUESTED** [review]

**Review by serge30 — APPROVED** [review]

**Pull request merged**